packages/compass-components/src/components/cancel-loader.tsx

```diff
@@ -27,29 +27,31 @@ const textDarkStyles = css({
 function CancelLoader({
   ['data-testid']: dataTestId = 'cancel-loader',
   progressText,
-  cancelText,
+  cancelText = 'Cancel',
   onCancel,
 }: {
   'data-testid'?: string;
   progressText: string;
-  cancelText: string;
-  onCancel: () => void;
+  cancelText?: string;
+  onCancel?: () => void;
```

Comment:
Yeah, I was thinking about moving label code there, but then there is not much left of cancel loader, I think I'll move label part to the spin loader, but then also just merge it all in one file with two exports so that it's not spread around so much